src/Legacy/Drupal7.php

```diff
@@ -43,7 +43,14 @@
 class Drupal7 {
 
   public function __call($method, $args) {
-    return call_user_func_array($method, $args);
+    // Hack: See http://goo.gl/aM5fra
```

Comment:
Reason: If I see a module with lots of: // Hack, etc.

I usually decide to not use it.